### PR TITLE
qOe4W1-genesis-validator-data

### PR DESCRIPTION
**Pull request by reza-bcw**
adding BCW keys

### CODE REVIEW

**Comment by raj-shekhar1** [comment]
@reza-bcw Can you re-check this, as the verify check is failing - 
```
./ethkey verifypop 0x6B5b2750A59ff6Fc7B087d21454761e4C33E218B enode://f84a192597597c98c6218ebab4ec10ad03fe7f69bc1920217562b71760a626d9163
9c0d2ef467ff06ee9210814b776b2392d06df9f230d52013fbfcaaccdcb72@35.214.255.214:30303 0x0916e717fcDBD625A99DfA468142D3EB251f6b5A 0xb30755eb557b8908b7ed46669b814189615fd7428de1c5d53eae7459aa34cbc308a7731abfeca02d2e2ee6dfa298f463 0x946feb58dd6ec3a2c3703e55686fa367c4cad3aea0ac92522bc2619287e910ac4b78f0f92b8ab2bcfe539fa37c7a8b0a34c0f4ba384ea14ae10755e8ea082fba0003bd31870ab239dd3b288ff13181edd4cb40c6b93e7fcfd687116f2a07d97ab82fa398fd99c886a39054e783795db413a3e049feaf119f91cb592314afe56d8d01896aeba86090fab324e81c76f671cfba0efaa489eebcc5aa71627b47fec42977e6c37c37063344078e94e8bec2dd2bbb19d200ad8df211ed280ada22430d6f85a34146b47e6e77312432f75c3d63efaa88609f2f640f6af39f6434cd971309de
===== POP Validation Results =====
Node Key:       false
Oracle Key:     true
Consensus Key:  false
```

**Comment by reza-bcw** [comment]
Hey raj thanks for currection. I changed the json. could you check it again?

File: genesis-validator-data/template.json
Comment (spenot09) [inline genesis-validator-data/template.json:1]: Heya, please avoid deleting this

**Review by spenot09 — APPROVED** [review]

**Comment by spenot09** [comment]
@raj-shekhar1 could you recheck this one please?

**Review by raj-shekhar1 — APPROVED** [review]
Looks good :+1: 
```
./ethkey verifypop 0x6B5b2750A59ff6Fc7B087d21454761e4C33E218B enode://f84a192597597c98c6218ebab4ec10ad03fe7f69bc1920217562b71760a626d91639c0d2ef467ff06ee9210814b776b2392d06df9f230d52013fbfcaaccdcb72@35.214.255.214:30303 0x0916e717fcDBD625A99DfA468142D3EB251f6b5A 0xb30755eb557b8908b7ed46669b814189615fd7428de1c5d53eae7459aa34cbc308a7731abfeca02d2e2ee6dfa298f463 0x4af47d9be17ab7a444c46bd5f0c206907ca6a52ad60c6f085053ede26982a89519fb8d3b000a771519d307c43c527bb0c28bf79fc5fa1bd7a2ebbeaf78861b5f0103bd31870ab239dd3b288ff13181edd4cb40c6b93e7fcfd687116f2a07d97ab82fa398fd99c886a39054e783795db413a3e049feaf119f91cb592314afe56d8d01ac25d2754cef1c5e685916f6d8aeb0ebe064d71e0981849793c5a43d5e6ae45a38123055ec53b3cffe20902577172dc618e06475a6165a0674b99c2e3520a5853ffbce1cd0c47604378efdfebaccd7f8e65013627adc1c85470065263f48b4bb
===== POP Validation Results =====
Node Key:       true
Oracle Key:     true
Consensus Key:  true
```

**Pull request merged**